tests/integration/element__scroll_to__test.py

```diff
@@ -22,20 +22,13 @@
 
 import os
 
-from selene.api.past import browser, config
-from selene.api.past import open_url
-from selene.support.jquery_style_selectors import s
 
 start_page = 'file://' + os.path.abspath(os.path.dirname(__file__)) + '/../resources/start_page.html'
 
 
-def setup_module(m):
-    config.browser_name = "chrome"
+def test_can_scroll_to(session_browser):
+    session_browser.open(start_page)
 
+    session_browser.element("#invisible_link").scroll_to().click()
 
-def test_can_scroll_to():
-    open_url(start_page)
-    # logging.warning(driver().current_url)
-    # driver().set_window_size(300, 400)
```

Comment:
@yashaka I deleted this code, but want to clarify some questions about it:
```
# logging.warning(driver().current_url)	
# driver().set_window_size(300, 400)
```
Do we need new test for low dimension? 
Or do we need to uncomment this code in this test?

Reply:
we need to uncomment, the idea was to have the browser size small enough so we can't see the needed element and have to scroll to it...

Reply:
Uncommented. Tested - window size is small and scroll works fine. Test passes.
Fixed in https://github.com/yashaka/selene/pull/237/commits/393c6a8e7c80b185df24fdb10003b9f9492eeeaa